crates/sdk/src/lib.rs

```diff
@@ -37,33 +37,28 @@ pub mod validation;
 pub mod error;
 pub mod events;
 pub(crate) mod internal_macros;
-pub mod io;
+
 #[cfg(feature = "migrations")]
 pub mod migrations;
 pub mod queries;
-pub mod task_env;
-pub mod wallet;
-
-#[cfg(feature = "async-send")]
-pub use std::marker::Send as MaybeSend;
-#[cfg(feature = "async-send")]
-pub use std::marker::Sync as MaybeSync;
 use std::path::PathBuf;
 use std::str::FromStr;
 
 use args::{DeviceTransport, InputAmount, SdkTypes};
-use io::Io;
-use masp::{ShieldedContext, ShieldedUtils};
 use namada_core::address::Address;
 use namada_core::collections::HashSet;
-pub use namada_core::control_flow;
 use namada_core::dec::Dec;
 use namada_core::ethereum_events::EthAddress;
 use namada_core::ibc::core::host::types::identifiers::{ChannelId, PortId};
 use namada_core::key::*;
 use namada_core::masp::{ExtendedSpendingKey, PaymentAddress, TransferSource};
+pub use namada_core::{control_flow, task_env};
+use namada_io::{Client, Io, NamadaIo};
```

Comment:
```suggestion
use namada_io::{Client, Io, NamadaIo};
pub use namada_io as io;
```
(this might need fmt)

Reply:
I think this should allow to undo lots of changes that use io from sdk